app/views/core/CreateAccountModal/teacher/SetupAccountPanel.coffee

```diff
@@ -9,6 +9,7 @@ SetupAccountPanel = Vue.extend
   mounted: ->
     @$store.dispatch('modal/createAccount')
     .catch (e) =>
+      window.tracker?.trackEvent 'CreateAccountModal Teacher SetupAccountPanel Error', category: 'Teachers', ['Mixpanel']
```

Comment:
Not sure we should capture errors in our analytics like this.  I don't think we do this anywhere else, so maybe a discussion of best practices is in order.